### PR TITLE
New module to setup error page body policies

### PR DESCRIPTION
**Pull request by grastogi23**
##### SUMMARY

New module to setup error page body policies

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

avi_errorpagebody

##### ANSIBLE VERSION

```
ansible 2.5.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = [u'/home/grastogi/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/grastogi/trace/ansible2.5/local/lib/python2.7/site-packages/ansible
  executable location = /home/grastogi/trace/ansible2.5/bin/ansible
  python version = 2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

File: lib/ansible/modules/network/avi/avi_errorpagebody.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+#
+# Created on Aug 25, 2016
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: supported
+#
+#
+# This file is part of Ansible
```
Comment (Akasurde) [inline lib/ansible/modules/network/avi/avi_errorpagebody.py]: Replace Line 9 to 23 with 

```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment (grastogi23) [reply lib/ansible/modules/network/avi/avi_errorpagebody.py]: The current license header was provided previously by Ansible Networking Group. I haven't seen any guideline on it. @gundalow any comments?

Comment (sivel) [reply lib/ansible/modules/network/avi/avi_errorpagebody.py]: This is documented at http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright

Comment (gundalow) [reply lib/ansible/modules/network/avi/avi_errorpagebody.py]: @grastogi23 It has changed since you created your initial modules. Please use the new form

Comment (grastogi23) [reply lib/ansible/modules/network/avi/avi_errorpagebody.py]: @sivel @gundalow Sure. I can change it. Also, would you like me to change it for all existing modules as well?

Comment (gundalow) [reply lib/ansible/modules/network/avi/avi_errorpagebody.py]: PR already merged which fixes the existing modules

**Pull request merged**